README.md

```diff
@@ -47,7 +48,8 @@ module "kinesis_firehose" {
 | tags | Map of tags to put on the resource | map | `null` | no |
 | cloudwatch_log_retention | Length in days to keep CloudWatch logs of Kinesis Firehose | integer | `30` | no |
 | log_stream_name | Name of the CloudWatch log stream for Kinesis Firehose CloudWatch log group | string | `SplunkDelivery` | no |
-| s3_bucket_name  | Name of the s3 bucket Kinesis Firehose uses for backups | string | `kinesis-firehose-to-splunk` | no |
+| s3_bucket_name  | Name of the s3 bucket Kinesis Firehose uses for backups | string | `kinesis-firehose-to-splunk` | yes |
```

Comment:
I realized a default s3 bucket name is not a good idea; it should be required input, as s3 buckets must be uniquely named across AWS.  I will remove the default.